components/ArtGenerator.js

```diff
@@ -0,0 +1,18 @@
+import styled from "styled-components";
+
+export default function ArtGenerator() {
+  return (
+    <section>
+      <Styledh2>Generator #1</Styledh2>
+      <StyledCanvas width="312px" height="312px"></StyledCanvas>
+    </section>
+  );
+}
+
+const StyledCanvas = styled.canvas`
```

Comment:
Here I don't mind combining the prefix `Styled` with the html tag name so much, because there's likely not going to be multiple canvases with different stylings.